source/adaptive_load/input_variable_setter_impl.cc

```diff
@@ -0,0 +1,38 @@
+#include "adaptive_load/input_variable_setter_impl.h"
+
+#include "envoy/registry/registry.h"
+
+#include "external/envoy/source/common/protobuf/protobuf.h"
+
+namespace Nighthawk {
+
+RequestsPerSecondInputVariableSetter::RequestsPerSecondInputVariableSetter(
+    const nighthawk::adaptive_load::RequestsPerSecondInputVariableSetterConfig&) {}
+
+void RequestsPerSecondInputVariableSetter::SetInputVariable(
+    nighthawk::client::CommandLineOptions& command_line_options, double input_value) {
+  command_line_options.mutable_requests_per_second()->set_value(
+      static_cast<unsigned int>(input_value));
+}
+
+std::string RequestsPerSecondInputVariableSetterConfigFactory::name() const {
+  return "nighthawk.rps";
+}
+
+Envoy::ProtobufTypes::MessagePtr
+RequestsPerSecondInputVariableSetterConfigFactory::createEmptyConfigProto() {
```

Comment:
I think this could have been a `const` method, but updating that probably requires changing the interface which already landed.

Reply:
Yes, opened #430